.github/workflows/combined_format_lint_test_mypy.yml

```
@@ -28,6 +27,104 @@ jobs:
           python -m pip install --upgrade pip
           pip install coverage mypy
           if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      
+      - id: files
+        name: Set up dependency for getting changed files
+        uses: jitterbit/get-changed-files@v1
+      
+      - name: Get all added or modified files
+        id: get_files
+        run: |
+          echo "FILES=$( echo ${{ steps.files.outputs.added_modified }} |
+            tr " " "\n" |
+            grep -E "\.py$" |
+            tr "\n" " ")" >> "$GITHUB_ENV"
+          echo "OTHER_FILES=$( echo ${{ steps.files.outputs.added_modified }} |
+            tr " " "\n" |
+            grep -E "\.txt$|\.json$|\.csv$"  |
```

```suggestion
            grep -E "^(?!.*\.py$).*"  |
```
I think we want to capture everything that is not a .py file, which could include a lot of different file types besides just txt, JSONs and CSVs.

---

Good call.